build/webmethods_traceability_agent.yml

```diff
@@ -8,7 +20,7 @@ webmethods_traceability_agent:
     deployment: ${CENTRAL_DEPLOYMENT:prod}
     environment: ${CENTRAL_ENVIRONMENT:""}
     environmentID: ${CENTRAL_ENVIRONMENTID} # only required when CENTRAL_USAGEREPORTING_OFFLINE=true
-    agentName: ${CENTRAL_AGENTNAME:""}
+    agentName: ${CENTRAL_AGENTNAME:"webmethods-agent-traceability"}
```

Comment:
This should be left empty and be overridden using the env var override. If not specified the SDK bootstrapping will create a agent resource using the environment name config

Reply:
fixed